tests/qam-updinstall/update_install.pm

```diff
@@ -208,7 +208,10 @@ sub run {
     foreach (@modules) {
         # substitue SLES_SAP for LTSS repo at this point is SAP ESPOS
         $_ =~ s/SAP_(\d+(-SP\d)?)/$1-LTSS/ if is_sle('15+');
-        $_ =~ s/SAP_(\d+(-SP\d)?)/SERVER_$1-LTSS/ if is_sle('=12-sp5') && !get_var('BUILD') =~ /saptune/;
+
```

Comment:
I am not sure this loop is doing anything useful in general. I think it should be removed.

Reply:
yes, you are right, in most case this loop is not useful, but I don't plan to modify it, it is better to create new ticket for this.

Reply:
I will remove the loop code in another PR

Reply:
Hi @a-kpappas, I checked some jobs on openQA, for example: https://openqa.suse.de/tests/16074456#settings,
I found that there are multiple links in INCIDENT_REPO,so the loop code for modules in update_install.pm may be needed, could you have any suggestion?